src/common/flags.ts

```diff
@@ -93,6 +93,20 @@ export const requiredDoceOrgFlag = OclifFlags.custom({
   required: true,
 });
 
+export const jobId = Flags.salesforceId({
+  char: 'i',
```

Comment:
You can add the Id prefix and the parse will validate it.

Reply:
Yes, it will.

Reply:
Standard Objects will have consistent prefixes in their ID's but Custom Objects (which our package installs) will not.